src/frontend-admin/src/components/reusables/HorizontalBarChart/HorizontalBarChart.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import styles from './HorizontalBarChart.module.css'
+
+//Prop Type
+interface IProps {
```

Comment:
Same comment here on the interface name. If you export this interface you can actually use it in your test file when you create your `props` constant.